eventuate-common-micronaut-id/src/main/java/io/eventuate/common/micronaut/id/IdGeneratorFactory.java

```diff
@@ -1,17 +1,26 @@
 package io.eventuate.common.micronaut.id;
 
+import io.eventuate.common.id.ApplicationIdGenerator;
 import io.eventuate.common.id.IdGenerator;
-import io.eventuate.common.id.IdGeneratorImpl;
+import io.eventuate.common.id.DatabaseIdGenerator;
 import io.micronaut.context.annotation.Factory;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.annotation.Value;
 
 import javax.inject.Singleton;
 
 @Factory
 public class IdGeneratorFactory {
 
   @Singleton
+  @Requires(missingProperty = "eventuatelocal.cdc.reader.id")
```

Comment:
Agree, but I used that name because we already have `eventuatelocal.cdc.reader.name` and I used a consistent property `eventuatelocal.cdc.reader.id`, because in general meaning it is a reader id. I will rename it.